README.md

```diff
@@ -7,173 +7,61 @@
 [![Mattermost](https://badgen.net/badge/Mattermost/Outline%20Community/blue)](https://community.internetfreedomfestival.org/community/channels/outline-community)
 [![Reddit](https://badgen.net/badge/Reddit/r%2Foutlinevpn/orange)](https://www.reddit.com/r/outlinevpn/)
 
-This is the repository to hold the future Outline SDK as we develop it. The goal is to clean up and move reusable code from [outline-ss-server](https://github.com/Jigsaw-Code/outline-ss-server) and [outline-go-tun2socks](https://github.com/Jigsaw-Code/outline-go-tun2socks).
+<p align="center">
+<img src="https://github.com/Jigsaw-Code/outline-brand/blob/main/assets/powered_by_outline/color/logo.png?raw=true" width=400pt />
+</p>
 
-**WARNING: This code is not ready to be used by the public. There's no guarantee of stability.**
+> **Warning**
+> This code is not ready to be used by the public. There's no guarantee of stability.
 
-Tentative roadmap:
+The Outline SDK helps developers:
+- Create tools to protect against network-level interference
+- Add network-level interference protection to existing apps, such as content or communication apps
+
+## Advantages
+
+| Multi-Platform | Proven Technology | Composable |
+|-|-|-|
+| The Outline SDK can be used on Android, iOS, Windows, macOS or Linux. | The Outline Client and Server have been using the SDK for years, helping millions of users in the harshest conditions. | The SDK interfaces were carefully designed to allow for composition and reuse, so you can craft your own transport. |
+
+## Integration
+
+The Outline SDK is written in Go. There are multiple ways to integrate the Outline SDK into your app:
+
+- As a **Go library**, in a Go application (CLI or graphical app with frameworks like [Fyne.io](https://fyne.io/), [Wails](https://wails.io/), [Qt for Go](https://therecipe.github.io/qt/), or [Go Mobile app](https://pkg.go.dev/golang.org/x/mobile/app)
```

Comment:
The Go GUI frameworks doesn't seem to be related to our SDK. Do we need to mention them here?

Reply:
People don't know how to do graphical apps in Go, so it's helpful to let them know about some possibilities.
Writing an app on those platforms may be easier than the other integration options.